### PR TITLE
Update Korean translation to b85d914

### PR DESCRIPTION
**Pull request by yous**
Also replaced '깃' with 'Git'.

### CODE REVIEW

File: README.ko.md [hunk 1]
```diff
@@ -630,21 +633,44 @@ $ hub clone tiimgreen/toc
...
+[풀 리퀘스트 템플릿 파일 생성기](https://www.talater.com/open-source-templates/)
+
 ### Octicons
 
 GitHub 아이콘(옥티콘)이 이제 오픈소스가 되었습니다.
 
 ![Octicons](https://og.github.com/octicons/octicons@1200x630.png)
 
-[*깃 허브의 옥티콘에 대해 더 읽어보세요.*](https://octicons.github.com)
+[*GitHub 옥티콘에 대해 더 읽어보세요.*](https://octicons.github.com)
```
Comment (marocchino) [inline README.ko.md]: @yous should we revert octicon?

**Comment by marocchino** [comment]
:+1:

File: README.ko.md (hunk 1 above)
Comment (yous) [inline README.ko.md]: Which one do you prefer? I'm neutral on this.

Comment (marocchino) [inline README.ko.md]: If you don't mind, I am okey with this.

Comment (yous) [inline README.ko.md]: Then I'll leave it now.

**Pull request merged**